### PR TITLE
clevo_nv41/fwupd_usage: Disable secure boot for fwupd

### PR DESCRIPTION
**Pull request by IgorBagnucki**
Signed-off-by: Igor Bagnucki <igor.bagnucki@3mdeb.com>

### CODE REVIEW

**Review by mkopec — APPROVED** [review]

File: docs/variants/clevo_nv41/fwupd_usage.md [hunk 1]
```diff
@@ -120,7 +120,9 @@ $ sudo ninja -C build install
 
 ## Updating firmware from LVFS
 
-1. Use the following command to update firmware with fwupd:
+1. Disable Secure Boot.
```
Comment (macpijan) [inline docs/variants/clevo_nv41/fwupd_usage.md:123]: @IgorBagnucki Can we elaborate how to do that? Or link to a document where we already have explained that?

Comment (macpijan) [reply docs/variants/clevo_nv41/fwupd_usage.md:123]: Also, the lockdown mode may be a bit different depending on a distro used.

How about also disabling lockdown as suggested here: https://github.com/flashrom/flashrom/issues/59 ?

Comment (mkopec) [reply docs/variants/clevo_nv41/fwupd_usage.md:123]: We don't have a specific procedure for disabling Secure Boot, closest we have is https://docs.dasharo.com/unified-test-documentation/dasharo-security/206-secure-boot/ which describes *enabling* Secure Boot, but the process is almost identical.

For disabling kernel lockdown, as far as i can tell the sysrq+x disable was never accepted upstream and only some kernels had it for some time, but dropped it due to security problems.

Comment (macpijan) [reply docs/variants/clevo_nv41/fwupd_usage.md:123]: @IgorBagnucki Let's link it then with the releveant description and then we can merge

Comment (IgorBagnucki) [reply docs/variants/clevo_nv41/fwupd_usage.md]: @macpijan The link is added.

**Pull request merged**